settings.yml

```diff
@@ -1,4 +1,5 @@
 port: "_env:PORT:3000"
+base-url: "_env:MIRIN_BASE_URL:cookpad.com"
```

Comment:
The default value doesn't contain http:// or https://. Is it valid for the location header?